R/plumber.R

```diff
@@ -890,29 +802,30 @@ plumber <- R6Class(
       filter <- PlumberFilter$new(name, expr, private$envir, serializer)
       private$addFilterInternal(filter)
     },
-    #' @description Retrieve openAPI file
-    swaggerFile = function() { #FIXME: test
-
-      swaggerPaths <- private$swaggerFileWalkMountsInternal(self)
-
-      # Extend the previously parsed settings with the endpoints
-      def <- utils::modifyList(private$globalSettings, list(paths = swaggerPaths))
-
-      # Lay those over the default globals so we ensure that the required fields
-      # (like API version) are satisfied.
+    #' @description Retrieve OpenAPI specification
+    openAPISpec = function() {
```

Comment:
I think it's a little shorter and less confusing with the caps

```suggestion
    apiSpec = function() {
```

In the documentation, we can always refer to it as the `OpenAPI Specification function`